common/conf/env/DEVINFRA.properties

```diff
@@ -121,6 +121,7 @@ pa.rugby.api.key=none
 
 # DCR
 rendering.endpoint=http://localhost:3030
+rendering.baseURL=http://localhost:3030
```

Comment:
Used by DotcomRenderingService for remote rendering, WSClient is mocked so no actual call should be made to this URL.